From 497db7e04b65882da99fc1d80fab1b015dd1da4f Mon Sep 17 00:00:00 2001
From: Anton Golub <antongolub@antongolub.com>
Date: Tue, 15 Aug 2023 13:21:10 +0300
Subject: [PATCH] test(cli): fix flaky test (#662)

---
 test/cli.test.js | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/test/cli.test.js
+++ b/test/cli.test.js
@@ -164,8 +164,10 @@
     `global {"_":["bar"]}\nimported {"_":["bar"]}\n`
   )
   assert.is(
-    (await $`node build/cli.js --eval 'console.log(argv._)' baz`).toString(),
-    `[ 'baz' ]\n`
+    (
+      await $`node build/cli.js --eval 'console.log(argv._.join(''))' baz`
+    ).toString(),
+    `baz\n`
   )
 })
 
